linux/deploy_vm/rebuild_ubuntu_unattend_install_iso.yml

```diff
@@ -0,0 +1,178 @@
+# Copyright 2022-2023 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Rebuild OS ISO image with unattended install file
+# Parameter:
+#   rebuilt_unattend_iso_path: Local path to the rebuilt ISO image with unattend install config file
+#
```

Comment:
Refer to review comments for rebuild_debian_unattend_install_iso.yml

Reply:
Thanks. Done